test/LibrarySearch/basic.lean

```diff
@@ -222,10 +222,13 @@ lemma prime_of_prime (n : ℕ) : Prime n ↔ Nat.Prime n := by
 -- Example from https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/exact.3F.20recent.20regression.3F/near/387691588
 lemma ex' (x : ℕ) (_h₁ : x = 0) (h : 2 * 2 ∣ x) : 2 ∣ x := by
   exact? says exact dvd_of_mul_left_dvd h
+mul_left_dvd h
 
+-- This example non-deterministically picks between `le_antisymm hxy hyx` and `ge_antisymm hyx hxy`.
+-- Commenting out for now.
 -- Example from https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Exact.3F.20fails.20on.20le_antisymm/near/388993167
-example {x y : ℝ} (hxy : x ≤ y) (hyx : y ≤ x) : x = y := by
-  exact? says exact le_antisymm hxy hyx
```

Comment:
Can we use `<|>` so that either suggestion is allowed?

Reply:
Oh sorry, I'm already commenting out the test in #7712. Should I take out of the queue?